Yafc.Parser/FactorioDataSource.cs

```diff
@@ -182,7 +182,7 @@ public static Project Parse(string factorioPath, string modPath, string projectP
                 versionSpecifiers = mods.mods.Where(x => x.enabled && !string.IsNullOrEmpty(x.version)).ToDictionary(x => x.name, x => Version.Parse(x.version!)); // null-forgiving: null version strings are filtered by the Where.
             }
             else {
-                allMods = new Dictionary<string, ModInfo> { { "base", null! }, { "elevated-rails", null! }, { "quality", null! }, { "space-age", null! } };
+                allMods = new Dictionary<string, ModInfo> { { "base", null! } };
```

Comment:
this might be a change that is too drastically
when space age DLC is installed and mod-list.json is missing, then the game loads space age

maybe we can move the `FindMods(factorioPath, progress, allFoundMods)` call above the calculation of allMods

Reply:
Added, now it only adds them when the DLC is available and mod-list.json is not found.

I also tried with a mod-list.json where they are disabled, and then they are (indeed) not loaded.